netbox/ipam/tables/vlans.py

```diff
@@ -65,9 +65,11 @@ class Meta(NetBoxTable.Meta):
         model = VLANGroup
         fields = (
             'pk', 'id', 'name', 'scope_type', 'scope', 'vid_ranges_list', 'vlan_count', 'slug', 'description',
-            'tags', 'created', 'last_updated', 'actions', 'utilization',
+            'tenant', 'tenant_group', 'tags', 'created', 'last_updated', 'actions', 'utilization',
+        )
+        default_columns = (
+            'pk', 'name', 'scope_type', 'scope', 'vlan_count', 'utilization', 'tenant', 'tenant_group', 'description'
```

Comment:
Suggest we omit `tenant_group` from the default column set (which I think is consistent with other tenant-enabled models).
```suggestion
            'pk', 'name', 'scope_type', 'scope', 'vlan_count', 'utilization', 'tenant', 'description',
```